src/analyses/apron/apronAnalysis.apron.ml

```diff
@@ -175,7 +206,14 @@ struct
 
   (* Function call transfer functions. *)
 
+  let pass_to_callee fundec reachable_from_args var =
+    let vname = Var.to_string var in
+    match List.find_opt (fun v -> v.vname = vname) (fundec.sformals @ fundec.slocals) with
+    | None -> true
+    | Some v -> Queries.LS.exists (fun (v',_) -> v'.vid = v.vid) reachable_from_args
```

Comment:
This seems a bit silly on a set. Do we need offsets of reachables for anything in the apron analysis? If not, then could just use a set of `varinfo`s and use `mem`.

Reply:
Going to a set of `varinfo`s is quite confusing though because one would need to account for `T`, so I think that this is the way clearer alternative, despite being a bit odd.

The alternative would be:

~~~OCaml
    let module VS = SetDomain.ToppedSet (CilType.Varinfo) (struct let topname = "All" end) in
    let reachable_vinfos e = try List.fold_left (fun s (v,_) -> VS.add v s) (VS.empty ()) (Queries.LS.elements (ctx.ask (ReachableFrom e))) with SetDomain.Unsupported _ -> VS.top ()  in
    let reachable_from_args = List.fold_left (fun ss s -> VS.union ss (rechable_vinfos s) ) (VS.empty ()) args in
~~~

Reply:
I guess it's fine for now as-is so we can quickly move to see if this helps us find any more relational invariants.

Totally off-topic, but `exists` on a topped set being `true` is actually weird, since it claims that `exists (fun _ -> false)` is also true :upside_down_face: